scripts/Dockerfile_test

```diff
@@ -1,8 +1,8 @@
 ARG image
 FROM $image
 
-COPY . datadog-lambda-ruby
-WORKDIR /datadog-lambda-ruby
+COPY . datadog-lambda-layer-ruby
```

Comment:
I renamed this to match the `-w` option definied in the docker run step of the `run_tests.sh`